pandas/io/parsers.py

```diff
@@ -338,9 +338,9 @@
     option can improve performance because there is no longer any I/O overhead.
 float_precision : str, optional
     Specifies which converter the C engine should use for floating-point
-    values. The options are `None` for the ordinary converter,
-    `high` for the high-precision converter, and `round_trip` for the
-    round-trip converter.
+    values. The options are `None` or `high` for the ordinary converter,
+    `legacy` for the original lower precision pandas converter, and
+    `round_trip` for the round-trip converter.
```

Comment:
@jreback doc string was updated already, so I think this is good to go??

Reply:
can you add a versionchanged tag for 1.2